search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -148,9 +149,15 @@
       {params :params headers :headers ctx :request-context query-string :query-string}
       ;; XXX REMOVE this check and the stubs once the service and
       ;;     the associations work is complete
-      (if (headers "cmr-prototype-umm")
-        (stubs/handle-prototype-request
-         path-w-extension params headers query-string)
+      (if (= "true" (string/lower-case (or (headers "cmr-prototype-umm") "")))
```

Comment:
NIT: use the existing `cmr.common.util/safe-lowercase` function instead of `string/lower-case`.

Reply:
Done.